client/src/ui/components/Cell.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Grid } from "@mui/material";
+import React from "react";
+import { useStyles } from "ui/styles";
+
+interface Props {
+  coordinates: { x: number; y: number };
+  gridValue: number;
+}
+
+const colors = ["black", "red"];
+
+const pickColor = (gridValue: number): string => {
+  return colors[gridValue - 1] || "transparent";
```

Comment:
NPE nebus? Netrowinantis get iš array yra arr.get(index), jis returnina undefined jei neranda elemento pagal index

Reply:
turėtu buti viskas ok, pasitikslinau su chatGPT katik, tai atrašė:
"The || operator is used to provide a default value of "transparent" if the first expression (colors[gridValue - 1]) evaluates to undefined."